lib/src/base-styles/_nx-text-input.scss

```
@@ -117,11 +117,25 @@
 
 .nx-text-input--textarea {
   .nx-text-input__box {
-    align-items: flex-end;
+    align-items: unset;
```

I'd feel more comfortable with `initial` here as unset feels more undefined in terms of what it would do
```suggestion
    align-items: initial;
```